kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorIT.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright [2020 - 2020] Confluent Inc.
+ */
+
+package io.confluent.connect.s3.integration;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.auth.policy.Policy;
+import com.amazonaws.auth.policy.Principal;
+import com.amazonaws.auth.policy.Statement;
+import com.amazonaws.auth.policy.actions.S3Actions;
+import com.amazonaws.auth.policy.resources.S3ObjectResource;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.CreateBucketRequest;
+import com.amazonaws.services.s3.model.ListVersionsRequest;
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import com.amazonaws.services.s3.model.S3VersionSummary;
+import com.amazonaws.services.s3.model.VersionListing;
+import io.confluent.common.utils.IntegrationTest;
+import io.confluent.connect.s3.S3SinkConnectorConfig;
+import io.confluent.testcontainers.squid.SquidProxy;
+import org.apache.kafka.connect.runtime.SinkConnectorConfig;
+import org.apache.kafka.connect.storage.StringConverter;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.kafka.connect.runtime.ConnectorConfig.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+@Category(IntegrationTest.class)
+public class S3SinkConnectorIT extends BaseConnectorIT {
+
+  private static final Logger log = LoggerFactory.getLogger(S3SinkConnectorIT.class);
+
+  private static final String CONNECTOR_NAME = "s3-sink-connector";
+  private static final long NUM_RECORDS_PRODUCED = 1000;
+  private static final int TASKS_MAX = 1;
+  private static final List<String> KAFKA_TOPICS = Arrays.asList("topic1");
+  private static final int FLUSH_SIZE = 200;
+  private int totalNoOfRecordsProduced = 0;
+  private static final String S3_TEST_BUCKET_NAME = "conn-dest-test-bucket";
```

Comment:
Why cant we use the `S3_BUCKET` defined in BaseConnectorIT?

Reply:
replaced